BraintreeUIKit/Public/BTUIKAppearance.h

```diff
@@ -47,6 +47,10 @@
 @property (nonatomic) UIKeyboardType postalCodeFormFieldKeyboardType;
 /// The highlighted version of the `tintColor`
 @property (nonatomic, readonly, getter = highlightedTintColor) UIColor *highlightedTintColor;
+/// Tint color for UISwitch when in the on position
+@property (nonatomic, strong) UIColor *switchOnTintColor;
```

Comment:
Do we want to have defaults for our light and dark themes for these values? See https://github.com/braintree/braintree-ios-drop-in/blob/master/BraintreeUIKit/Helpers/BTUIKAppearance.m#L38

Reply:
I think the default iOS colors look fine on light mode. Do we want to change anything on dark mode?

![Light mode](https://user-images.githubusercontent.com/35243507/58184296-773d4500-7c76-11e9-9c62-8900abe36fe5.png)
![Dark mode](https://user-images.githubusercontent.com/35243507/58184307-799f9f00-7c76-11e9-86b1-9dddcf4aba20.png)